angular/projects/spark-angular/src/environment/environment.ts

```diff
@@ -1,3 +1,3 @@
 export const environment = {
-  version: '14.0.0',
+  version: '14.0.0'
```

Comment:
I removed this file in my angular storybook upgrade PR so we won't have to deal with this anymore.